packages/experimental/Avatar/src/JSAvatar.styling.ts

```diff
@@ -95,7 +95,6 @@ export const stylingSettings: UseStylingOptions<JSAvatarProps, AvatarSlotProps,
         return {
           style: {
             borderStyle: 'solid',
-            borderColor: tokens.ringColor,
             borderWidth: ringConfig.stroke,
```

Comment:
Just curious, should this take borderWidth instead of ringConfig.stroke?

Reply:
I think yes. I still plan to review this ring object to see what's we really need from it